mwapi/errors.py

```diff
@@ -36,3 +48,38 @@ class LoginError(RuntimeError):
     @classmethod
     def from_doc(cls, doc):
         return cls(doc.get('result'))
+
+
+class RequestError(requests.exceptions.RequestException):
```

Comment:
Can't we just have a HTTPError and inside that wrap the underlying exception from requests instead of having one for each?